tests/test_api.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+import requests
+from proton import cert_pinning
+
+class TestCertificatePinning():
+    
+    s = requests.Session()
+    
+    def test_api_url(self):
```

Comment:
The best practice for this is to have two tests, one testing the status `200` and the other checking that the exception is thrown.
Also, to assert that an exception is thrown, you should use the construct `with pytest.raises`, check https://docs.pytest.org/en/stable/assert.html 😉